### PR TITLE
Add readme about installation & use of build/push scripts

### PR DESCRIPTION
**Pull request by serchaos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Stevedore 
+
+Stevedore is a helpful aid(e) in building and deploying Docker images to container registries, assuming your code is hosted in a git repository. 
+It encompasses: 
+ - A build script to build and tag images
+ - A push script to build, tag, and then push images to a repository 
+ - A standalone server to poll git repositories for changes, then build, tag, and push images to a repository
```
Comment (smreed) [inline README.md]: This version doesn't have the "server" anymore.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## Stevedore 
+
+Stevedore is a helpful aid(e) in building and deploying Docker images to container registries, assuming your code is hosted in a git repository. 
+It encompasses: 
+ - A build script to build and tag images
+ - A push script to build, tag, and then push images to a repository 
+ - A standalone server to poll git repositories for changes, then build, tag, and push images to a repository 
+
+## Using it
```
Comment (smreed) [inline README.md]: Actually, it's `go get`-able. `go get github.com/zulily/stevedore/...`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The easiest way to use the build & push scripts is to clone the source of this repository, navigate to the individual stevedore-build/ and stevedore-push/ directories, and then `go install .` on each.
+That puts them in your $GOBIN (provided it's set), and you should be able to call them directly from there.
+
+## Build script 
+
+Inside the stevedore-build folder is the code for building images locally.  
+Available command-line parameters: 
+ - `-verbose`: enables verbose output
+ - `-i [PATTERN]`: only build Dockerfiles with file extensions that match the regex [PATTERN], e.g. `stevedore-build -i dev`
```
Comment (smreed) [inline README.md]: You can also explicitly specify the dockerfiles as arguments to the commands (I prefer this to the regexp actually)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+When building, Stevedore automatically tags your built image with [REGISTRY]:[GIT_SHA].
+
+## Push script
+
+The push script (the stevedore-push folder) contains code for building as well as pushing.
+It accepts all of the same arguments that the build script does. 
+
+In addition to building and tagging with the GIT_SHA of the repo's current HEAD, it will also build and tag a secondary image with `latest`.
+
+## Standalone server
```
Comment (smreed) [inline README.md]: Remove

File: README.md (hunk 2 above)
Comment (serchaos) [inline README.md]: Oh, cool. Much more concise.

File: README.md (hunk 1 above)
Comment (serchaos) [inline README.md]: That's what I get for documenting from memory. Removing.

Comment (smreed) [inline README.md]: ping me when you push the next commits up

**Comment by smreed** [comment]
LGTM, merging...

**Pull request merged**